specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineExamples/VirtualMachine_Get_VM_WithInstanceView.json

```diff
@@ -0,0 +1,220 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2022-08-01",
+    "vmName": "myVM",
+    "$expand": "userData,instanceView"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "myVM",
+        "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachines/myVM",
+        "type": "Microsoft.Compute/virtualMachines",
+        "location": "West US",
+        "tags": {
+          "myTag1": "tagValue1"
+        },
+        "properties": {
+          "vmId": "0f47b100-583c-48e3-a4c0-aefc2c9bbcc1",
+          "availabilitySet": {
+            "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/availabilitySets/my-AvailabilitySet"
+          },
+          "proximityPlacementGroup": {
+            "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/proximityPlacementGroups/my-ppg01"
+          },
+          "hardwareProfile": {
+            "vmSize": "Standard_DS3_v2"
+          },
+          "storageProfile": {
```

Comment:
add the diskControllerType property too with the value of SCSI.

Reply:
acknowledging